EF-Adapter/CasbinDbAdapter.cs

```diff
@@ -20,17 +20,32 @@ public CasbinDbAdapter(CasbinDbContext context)
 
         #region sync operations
 
+        /// <inheritdoc />
         public void LoadPolicy(NetCasbin.Model.Model model)
         {
             var rules = _context.CasbinRule.AsNoTracking().ToList();
             LoadPolicyData(model, Helper.LoadPolicyLine, rules);
         }
-        
+
+        /// <inheritdoc />
         public void RemovePolicy(string sec, string ptype, IList<string> rule)
         {
             RemoveFilteredPolicy(sec, ptype, 0, rule.ToArray());
         }
 
+        /// <inheritdoc />
+        public void AddPolicies(string sec, string ptype, IEnumerable<IList<string>> rules)
+        {
+            Task.FromResult(AddPoliciesAsync(sec, ptype, rules));
```

Comment:
The difference between the sync and async method is to use SaveChanges or SaveChanegsAsync, we should not create unnecessary task instances.